### PR TITLE
Update betaflight-configurator_10.8.0_amd64.deb to 10.9.0

### PR DESCRIPTION
**Pull request by razzeee**

### CODE REVIEW

**Comment by jimmac** [comment]
Can confirm the build functioning. Connection can be established when plugged in prior to launching.
Screenshot from 2023-02-03 20-32-08

File: io.github.betaflight.BetaflightConfigurator.appdata.xml [hunk 1]
```diff
@@ -35,23 +35,24 @@
...
       <caption>Drone setup screen</caption>
     </screenshot>
     <screenshot>
-      <image>https://raw.githubusercontent.com/razzeee/io.github.betaflight.BetaflightConfigurator/update-deb/screenshots/Betaflight3.png</image>
+      <image>https://raw.githubusercontent.com/flathub/io.github.betaflight.BetaflightConfigurator/master/screenshots/Betaflight3.png</image>
       <caption>Motor configuration screen</caption>
     </screenshot>
     <screenshot>
-      <image>https://raw.githubusercontent.com/razzeee/io.github.betaflight.BetaflightConfigurator/update-deb/screenshots/Betaflight4.png</image>
+      <image>https://raw.githubusercontent.com/flathub/io.github.betaflight.BetaflightConfigurator/master/screenshots/Betaflight4.png</image>
```
Comment (hfiguiere) [inline io.github.betaflight.BetaflightConfigurator.appdata.xml]: don't link to master like that. Link to a specific commit.

Comment (razzeee) [reply io.github.betaflight.BetaflightConfigurator.appdata.xml]: Good call, changed

**Comment by Canaill51** [comment]
Hello! Link on Flathub is not up to date yet. Is this normal?
Looking your last comments here, I understand that 10.9.0 is ready to use as Flatpak.
Could you please confirm?
Thx :)

**Comment by razzeee** [comment]
It needs to be merged here first, to be available. I don't have those rights.

**Comment by hfiguiere** [comment]
@barthalion maybe you can invite @razzeee ?

I'll merge it after this test build.

bot, build

**Pull request merged**